### PR TITLE
Add docs for say.py to indicate how to list voices

### PR DESCRIPTION
**Pull request by smolz**
##### SUMMARY

Added a line that describes where to find the available voices for the command

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

say.py
##### ANSIBLE VERSION

```
ansible 2.5.0rc2
  config file = None
  configured module search path = [u'/Users/smolz/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/smolz/Dropbox/Programming/python-virtualenvs/ansible-2.5.0.0/lib/python2.7/site-packages/ansible
  executable location = /Users/smolz/Dropbox/Programming/python-virtualenvs/ansible-2.5.0.0/bin/ansible
  python version = 2.7.14 (default, Mar  9 2018, 23:57:12) [GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: lib/ansible/modules/notification/say.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 notes:
    - In 2.5, this module has been renamed from M(osx_say) into M(say).
    - If you like this module, you may also be interested in the osx_say callback plugin.
+   - A list of available voices, with language, can be found by running `say -v ?` on a OSX computer.
```
Comment (angystardust) [inline lib/ansible/modules/notification/say.py]: you can also add: "and `espeak --voices` on a Linux host."

**Review by smolz — COMMENTED** [review]
added linux command as well

File: lib/ansible/modules/notification/say.py [hunk 2]
```diff
@@ -23,6 +23,7 @@
 notes:
    - In 2.5, this module has been renamed from M(osx_say) into M(say).
    - If you like this module, you may also be interested in the osx_say callback plugin.
+   - A list of available voices, with language, can be found by running `say -v ?` on a OSX host and `espeak --voices` on a Linux host.
```
Comment (sivel) [inline lib/ansible/modules/notification/say.py]: Instead of backticks, can you wrap with `C()`?

**Review by smolz — COMMENTED** [review]
Changed backticks to C()

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
@smolz Thank you for this. I've merged it into `devel` and created a backport PR for `stable-2.7`

**Pull request merged**